firefox-desktop/views/mobile_forecasts.view.lkml

```diff
@@ -0,0 +1,162 @@
+# note that to get the union to work I had to re-order the columns of the fx_ios forecast table, for some reason they are in a different order.
+view: mobile_forecasts {
+  derived_table: {
+    sql: WITH base as (
```

Comment:
It's probably worth it to push these to a BQ view. I won't block on this but we should open an issue.

Reply:
I agree. That issue should also involve moving these tables into a static dataset that is only writeable by ops and DE rather than `analysis`.